### ISSUE TITLE
any sfpowerkit:pool command gives false warning "ScratchOrgInfo.Allocation_status__c field is missing"

### ISSUE THREAD

**Issue by busybox0** [dxatscale/sfpowerkit#685]
**Describe the bug**
any sfpowerkit:pool command gives false warning:

```
Required Prerequisite values in ScratchOrgInfo.Allocation_status__c field is missing in the DevHub, expected values are : In Progress,Available,Allocate,Assigned
Switching back to previous version, we request you to update ScratchOrgInfo.Allocation_status__c field in the DevHub 
For more information Please refer https://github.com/Accenture/sfpowerkit/blob/main/src_saleforce_packages/scratchorgpool/force-app/main/default/objects/ScratchOrgInfo/fields/Allocation_status__c.field-meta.xml 
```
**To Reproduce**
```
$ sfdx sfpowerkit:pool:list  -v DevHub
-------------------------------------------------------------------------------------------
sfpowerkit  -- The DX@Scale Developer Toolkit - Version:4.2.8 - Release:May 22
-------------------------------------------------------------------------------------------
Required Prerequisite values in ScratchOrgInfo.Allocation_status__c field is missing in the DevHub, expected values are : In Progress,Available,Allocate,Assigned
Switching back to previous version, we request you to update ScratchOrgInfo.Allocation_status__c field in the DevHub 
For more information Please refer https://github.com/Accenture/sfpowerkit/blob/main/src_saleforce_packages/scratchorgpool/force-app/main/default/objects/ScratchOrgInfo/fields/Allocation_status__c.field-meta.xml 

======== Scratch org Details ========
List of all the pools in the org
TAG         COUNT
──────────  ─────
sfpowerkit  4
===================================
Unused Scratch Orgs in the Pool : 4 

TAG         ORG ID           USERNAME                       EXPIRY DATE  STATUS     LOGIN U R L
──────────  ───────────────  ─────────────────────────────  ───────────  ─────────  ───────────────────────────────────────────────────────
< orgs details here >
```

**Background**
package 04t1P000000katQQAQ  was installed to devHub (following [instruction on this link](https://github.com/dxatscale/sfpower-scratchorg-pool) ), which shows as  **sfpower-scratchorg-pool | Accenture Australia | 2.0**
The field ScratchOrgInfo.Allocation_status__c is in org and is editable by DevHub user, along with all other neighbor fields.

**Screenshots**

![Screenshot 2022-06-27 191623](https://user-images.githubusercontent.com/6568234/175988957-12f8def1-3459-4e57-93ca-9b962272c0b8.png)

![Screenshot 2022-06-27 191924](https://user-images.githubusercontent.com/6568234/175989000-2bc9101a-c80b-499f-b2d8-9c763368dc44.png)

here is Scratch Org Info Custom Field   **Allocation status** (Unlocked)
![Screenshot 2022-06-27 192113](https://user-images.githubusercontent.com/6568234/175989040-c674add9-033a-403c-bf1b-d1515d22cc67.png)

**Desktop:**
-   OS: **Both** Windows/Linux
-   sfdx-cli/7.154.0 wsl-x64 node-v14.17.6

**Comment by Caitlyn-Mills** [comment]
Will push a fix for this on Thursday

**Comment by busybox0** [comment]
Going deeper in investigations, there is more than cosmetic warning, Seems that scheduled run of sfpowerkit:pool:create  ignores orgs withdrawn by  sfpowerkit:pool:fetch  and refuses to top up the pool count.

**Comment by Caitlyn-Mills** [comment]
This issue is fixed in 4.2.11 
Thanks!
